tools/internal_ci/linux/grpc_bazel_rbe.sh

```diff
@@ -34,3 +34,7 @@ bazel_rbe/bazel_wrapper \
   $BAZEL_FLAGS \
   "$@" \
   -- ${BAZEL_TESTS:-//test/...}
+
+# Print disk usage
+df -h
+du -hs * | sort -rh | head -100
```

Comment:
`du -h --max-depth=1` will also capture hidden dirs. `.cache` and `.local` on my systems are generally always hefty, unclear what the build environment may be hiding where. It'll be interesting to see if the repo root takes up the majority of the space, too.

Reply:
but with max-depth, we won't get subdirectories correct?